koopovereenkomst-v3-simple/pages/brk.tsx

```diff
@@ -0,0 +1,55 @@
+import { useSession } from "@inrupt/solid-ui-react";
+import { Button } from "@mui/material";
+import { Box } from "@mui/system";
+import Head from "next/head";
+import { useEffect, useState } from "react";
+import ConnectSolid from "../src/ConnectSolid";
+import Layout from "../src/Layout";
+import VC, { VCType } from "../src/VC";
+
+export default function Brk() {
```

Comment:
Is er ook een manier om terug te gaan naar de koek flow? Of moet je dan Back in de browser gaan?

Reply:
Close venster ... in goed NL 😆 De 'BRK' wordt in een nieuw tabblad geopend ...

Reply:
Works! 💪